### ISSUE TITLE
Special case where packs are split

### ISSUE THREAD

**Issue by timcu** [timcu/sas-select#6]
For one particular part (Omnigon 28300), the pack size is 10 but Qld Stoma Association splits it into 2 so patients can order 5.

**Comment by chayawit** [comment]
Therefore, Entitlement in this case can be hardcoded as 0.5?

**Comment by timcu** [comment]
I am open to suggestions on how to handle this. I would create a dict of special cases. There is currently only one special case. The dict will contain QSA pack size, eg { ('28300', 'Omnigon Bbraun Urimed Bag 2L'): 5 }. Hence they can order one QSA pack of 5 rather than manufacturers pack of 10.  However, your suggestion to allow order fractional packs for this product may also work.

**Comment by chayawit** [comment]
Are SAS Codes unique? I'm thinking to use them as keys of the dict.

**Comment by timcu** [comment]
The combination of Company Code and Brand is unique. Two Brands might use the same Company Code for different products. None of the other fields are unique.

**Comment by chayawit** [comment]
Hi, Tim. Review this snippet for me please. Thank you.
```py
        special_entitlements = {('28300', 'Omnigon Bbraun Urimed Bag 2L'): {'packs': 0.5, 'frequency': 'month'}}
        unique_product = (comp_code, brand)
        if unique_product in special_entitlements:
            packs = special_entitlements[unique_product]['packs']
            frequency = special_entitlements[unique_product]['frequency']
            return "Your medicare entitlement is " + str(packs) + " packs per " + frequency
```

**Comment by timcu** [comment]
Looks good to me. Did you test it?

**Comment by chayawit** [comment]
Thanks, Tim. Yes, I've tested. Will commit soon.

**Comment by timcu** [comment]
Fixed in PR #37
